agenta-web/src/components/Playground/VersionTabs.tsx

```diff
@@ -146,6 +146,22 @@ const VersionTabs: React.FC = () => {
         })
     }
 
+    // Map the variants array to create the items array conforming to the Tab interface
+    const tabItems: PlaygroundTabsItem[] = variants.map((variant, index) => ({
+        key: variant.variantName,
+        label: `Variant ${variant.variantName}`,
+        children: (
+        <ViewNavigation
+            variant={variant}
+            handlePersistVariant={handlePersistVariant}
+            setRemovalVariantName={setRemovalVariantName}
+            setRemovalWarningModalOpen={setRemovalWarningModalOpen2}
+            isDeleteLoading={isDeleteLoading}
+        />
+        ),
+        closable: !variant.persistent,
+    }));
```

Comment:
**Suggestion:** Consider adding a type to the `handlePersistVariant` function. This will help ensure that the function is used correctly and will make the code easier to understand.
```suggestion
    const tabItems: PlaygroundTabsItem[] = variants.map((variant: Variant, index: number) => ({
        key: variant.variantName,
        label: `Variant ${variant.variantName}`,
        children: (
        <ViewNavigation
            variant={variant}
            handlePersistVariant={handlePersistVariant as (variant: Variant) => void}
            setRemovalVariantName={setRemovalVariantName}
            setRemovalWarningModalOpen={setRemovalWarningModalOpen2}
            isDeleteLoading={isDeleteLoading}
        />
        ),
        closable: !variant.persistent,
    }));
```